src/CoreWCF.Primitives/src/CoreWCF/Dispatcher/OperationInvokerBehavior.cs

```diff
@@ -12,8 +14,44 @@ public class OperationInvokerBehavior : IOperationBehavior
     {
         private IServiceProvider _serviceProvider;
 
+        private static readonly Lazy<bool> s_isOperationInvokerGeneratorEnabled = new Lazy<bool>(() =>
+        {
+            if (AppContext.TryGetSwitch("CoreWCF.Dispatcher.UseGeneratedOperationInvokers", out var value))
+            {
+                return value;
+            }
+
+            Assembly entryAssembly = Assembly.GetEntryAssembly();
+            // return null when running UT with Xunit and .NET Framework
+            if (entryAssembly != null)
+            {
+                foreach (var attribute in entryAssembly.GetCustomAttributesData())
+                {
+                    if (attribute.AttributeType == typeof(EnableCoreWCFOperationInvokerGeneratorAttribute))
+                    {
+                        foreach (var argument in attribute.NamedArguments)
+                        {
+                            if (
+                                argument.TypedValue.ArgumentType == typeof(string)
```

Comment:
An easier way to see if this is the correct parameter is to do:
```c#
if (argument.MemberName == "Value")
{
    var stringValue = argument.TypedValue.Value as string;
    return string.Equals(stringValue, "true", StringComparison.OrdindalIgnoreCase);
}
```
This code only works if you use `[EnableCoreWCFOperationInvokerGenerator(Value = "true")]`, but you've also added a constructor parameter so you can also use `[EnableCoreWCFOperationInvokerGenerator("true")]` which this code won't work for. After completing the foreach around the NamedArguments, you also need to iterate over ConstructorArguments. Alternatively you could limit setting it to just one way, so remove the constructor argument from the class and require it be set via the named property as you've already got this code. You could also switch to only using ConstructorArguments and remove the property. We don't need to actually store the value as when using CustomAttributesData, you're reading out the code declaration and not actually instantiating an instance of the attribute.

Reply:
updated to use a ctor boolean arg. I reproduced what is done with common atributes like FileVersionAttribute.
```c#
namespace System.Reflection
{
    [AttributeUsage(AttributeTargets.Assembly, Inherited = false)]
    public sealed class AssemblyFileVersionAttribute : Attribute
    {
        public AssemblyFileVersionAttribute(string version)
        {
            Version = version ?? throw new ArgumentNullException(nameof(version));
        }

        public string Version { get; }
    }
}
```